staging/src/k8s.io/component-base/metrics/legacyregistry/registry.go

```diff
@@ -83,3 +91,8 @@ func CustomMustRegister(cs ...metrics.StableCollector) {
 		prometheus.MustRegister(c)
 	}
 }
+
+// Reset calls reset on the global registry
```

Comment:
Sometimes I do `var Reset = defaultRegistry.Reset()` instead of a function, to make this even more obvious (up to you).

Reply:
ah nice, yeah i made the refactoring for that and some other code sites where that comment is applicable.